src/Entity/User.php

```diff
@@ -32,41 +37,56 @@ class User implements UserInterface
     /**
      * @var string The hashed password
      * @ORM\Column(type="string")
-     */
+    */
     private $password;
 
     /**
-     * @ORM\Column(type="string", length=255)
+     * @ORM\Column(type="string", length=40)
```

Comment:
Il ne faut pas changer les valeurs dans les annotations sinon ça va changer à la prochaine migration, et 255 car sont nécessaires pour le password hashé

Reply:
pour le password ça n'a pas été modifié justement

Reply:
c'est la ligne au dessus :    37 a 41 
/**
     * @var string The hashed password
     * @ORM\Column(type="string")
    */
    private $password;